Classes/ViewHelpers/Page/LinkCSSViewHelper.php

```diff
@@ -58,11 +59,21 @@ public static function renderStatic(
         \Closure $renderChildrenClosure,
         RenderingContextInterface $renderingContext
     ) {
-        $fileNameFromArguments = $arguments['file'];
-        if ($fileNameFromArguments) {
-            $CSSFileName = GeneralUtility::makeInstance(FilePathSanitizer::class)->sanitize($fileNameFromArguments);
-            $pageRenderer = GeneralUtility::makeInstance(PageRenderer::class);
-            $pageRenderer->addCSSFile($CSSFileName);
+        $typo3VersionConstraint = version_compare(VersionNumberUtility::getNumericTypo3Version(), '10.0.0', '<');
```

Comment:
In case the deprecation message is already shown in 9.5.x and the `FileSanitizer` already exists in 9.5.x, the comparison should be against 9.5.0